test/unit/parse_conditions_test.lua

```diff
@@ -101,3 +102,27 @@ g.test_parse_errors = function()
         'condition[2] should be string, got "number" (condition 2)'
     )
 end
+
+g.test_jsonpath_parse = function()
+    t.skip_if(not crud_utils.tarantool_supports_jsonpaths(), "Jsonpath is not supported on Tarantool < 2")
```

Comment:
It's filters it's available since 1.10. But you don't have indexes with "path" - see https://github.com/tarantool/doc/issues/708

Reply:
Here is a doc ticket about json-path access inside tuple - https://github.com/tarantool/doc/issues/501